test/browser/features/plugin_angular.feature

```diff
@@ -1,7 +1,7 @@
 @plugin_angular
 
 # Skipped on older Safari versions not supported by Angular 10 - Angular renders the fixture component twice, causing duplicate events to be reported
-@skip_safari_10 @skip_ios_10
+@skip_safari_10 @skip_below_ios_13
```

Comment:
For this to work you have to implement the tag `@skip_below_ios_13` yourself, there's no magic behind it.  See `skip.rb` for the definitions of them.

Reply:
there is an implementation present in `packages/react-native/ios/vendor/bugsnag-cocoa/features/support/env.rb` which I thought would be used?